### PR TITLE
ADCC-4131 Bump clickhouse-jdbc version

### PR DESCRIPTION
**Pull request by denissinkov**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -51,38 +35,95 @@
...
+                    <id>ossrh</id>
+                    <url>https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+                </repository>
+            </distributionManagement>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-source-plugin</artifactId>
+                        <version>3.2.1</version>
```
Comment (iamlapa) [inline pom.xml]: Consider extracting library versions into properties in the `pom.xml` file.

File: pom.xml [hunk 2]
```diff
@@ -51,38 +35,95 @@
...
+                                <goals>
+                                    <goal>jar-no-fork</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <version>3.2.0</version>
```
Comment (iamlapa) [inline pom.xml]: Consider extracting library versions into properties in the `pom.xml` file.

File: pom.xml [hunk 3]
```diff
@@ -51,38 +35,95 @@
...
+                                    <keyname>${gpg.keyname}</keyname>
+                                    <passphraseServerId>${gpg.keyname}</passphraseServerId>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.sonatype.plugins</groupId>
+                        <artifactId>nexus-staging-maven-plugin</artifactId>
+                        <version>1.6.8</version>
```
Comment (iamlapa) [inline pom.xml]: Consider extracting library versions into properties in the `pom.xml` file.

File: pom.xml (hunk 1 above)
Comment (denissinkov) [reply pom.xml]: fixed

File: pom.xml (hunk 2 above)
Comment (denissinkov) [reply pom.xml]: fixed

File: pom.xml (hunk 3 above)
Comment (denissinkov) [reply pom.xml]: fixed

**Review by iamlapa — APPROVED** [review]

**Pull request merged**